website/docs/blueprints/data-analytics/datahub-on-eks.md

```diff
@@ -78,7 +78,7 @@ chmod +x install.sh
 
 ### Verify Deployment
 
-After the deployment completes, we can access the DataHub UI and test importing metadata from sample datasources.  For demo purpose, this blueprint creates the Ingress object for the datahub FrontEnd UI with public LoadBalancer(internet-facing).  For production workloads, you can modify datahub_values.yaml to use internal LB:
+After the deployment completes, we can access the DataHub UI and test importing metadata from sample datasources.  For demo purpose, this blueprint creates the Ingress object for the datahub FrontEnd UI with public LoadBalancer(internal # Private Load Balancer can only be accessed within the VPC).  For production workloads, you can modify datahub_values.yaml to use internal LB:
```

Comment:
This was probably a search/replace, so this documentation got changed in an odd manner.

Reply:
Thank you👍 We will update the docs